package.json

```diff
@@ -28,6 +29,9 @@
 	"dependencies": {
 		"@zip.js/zip.js": "2.7.20",
 		"joplin-turndown-plugin-gfm": "1.0.12",
+		"pako": "^2.1.0",
```

Comment:
Thanks, this shaves quite a bit off the bundle size and makes it a bit faster (since it's doing the gunzipping in native code not JS I presume). Also it's good to have less dependencies.